packages/blue-sdk/src/market/Market.ts

```diff
@@ -273,11 +265,7 @@ export class Market implements InputMarket {
     return { market, assets, shares };
   }
 
-  public borrow(
-    assets: bigint,
-    shares: bigint,
-    timestamp: bigint = this.lastUpdate,
-  ) {
+  public borrow(assets: bigint, shares: bigint, timestamp: BigIntish) {
```

Comment:
Remove default value which is misleading: it could be the last market's update, the current timestamp, based on the use case. Prefer to not give defaults and force the dev to provide the value, to think on the use case